cogs/challenges.py

```diff
@@ -194,10 +204,23 @@ async def on_message(self, message):  # Submitted role.
                 embed.set_footer(
                     text=f"#ID: {message.author.id} • {len(code)} chars • Language: {filetype}"
                 )
+                self.code_length[message.author.id] = (filetype, len(code))
+                
                 await submission_channel.send(embed=embed)
 
         elif message.channel.id in [
             680851798340272141,
             713841395965624490,
         ]:  # Automatic reaction
             await message.add_reaction("🖐️")
+        elif message == 't.char':
```

Comment:
Make this a normal command.

Reply:
the normal commands can only be accessed by people with the challenge host, challenge host helper or staff role but we want members to use this command too

Reply:
That's just the for the challenge group.